_posts/2018-05-25-support-process.markdown

```diff
@@ -0,0 +1,99 @@
+
+Out of this effort, we learned that people had some shared concerns:
+
+* The same few people tended to address most of the support questions, meaning the responsibility was imbalanced.
+* People felt wary about the idea of having to answer to support requests from areas they were unfamiliar with.
+* People wondered how a support duty might fit into their product work.
+* Without a dedicated process people worried that even if bugs were identified they might not be fixed (thus contributing to future support needs).
+
+We then identified a few potential solutions:
+
+* We could hire or dedicate people to handle support as a full-time job.
+* We could have a rotating set of engineers who are exempt from product-related work and tasked with both answering immediate support requests and fixing small bugs or improving the state of our infrastructure/monitoring.
+* We could have engineers on an on-call rotation who are responsible for addressing immediate issues but remain engaged with their product team.
+
+# Our Plan
+
+We worked with the product management team to identify an escalation path for different types of issues. At the least-urgent level were basic feature requests, and at the most-urgent were critical bugs or “incidents”. We decided to focus on building out a process whereby certain engineers could be available to address urgent incidents, and other, less-urgent bugs or issues could be triaged and added to teams’ workloads as part of a normal sprint.
+
+Out of the solutions above, we chose the on-call approach where engineers remain part of product sprints but prioritize responding to critical issues. We felt it was important for on-call engineers to stay involved with their team’s ongoing work and also to encourage everyone to share responsibility for our system health.
+
+In order to make the idea of an “on-call” shift concrete and get feedback from the entire engineering team, we put a document up for review that describes the “support plan” in detail. Here’s an abbreviated version of that document:
+
+## Process Overview
+
+Two engineers are on-call each week. The rotating schedule is published in a Google calendar at least a month in advance and engineers are encouraged to trade shifts as necessary.
+
+During work hours, on-call engineers are responsible for responding to issues in our #incidents slack channel. Outside of work hours, they are only responsible for downtime issues.
+
+## On-Call Resonsibilities
+
+While on-call, you are accountable for investigating and fixing timely issues, escalating to additional point-people and/or routing to team-specific backlogs where appropriate.
+
+Your top priority during an on-call period is to address critical issues. Use your judgment about joining ongoing sprint and team activities.
+
+1.  Be available to answer requests and respond to immediate issues.
+2.  Investigate and address critical issues using shared playbooks and pulling in point-people where necessary.
```

Comment:
I wonder if the phrase "shared playbooks" could be simplified to "documentation". You and I know what a playbook means in the context of Artsy, but maybe it's something that wouldn't be understood universally? For example, I hadn't heard of playbooks before coming to Artsy.